rwengine/src/script/ScriptMachine.cpp

```diff
@@ -32,7 +32,6 @@ void ScriptMachine::interuptNext()
     interupt = true;
 }
 
-#include <iostream>
```

Comment:
nit: Might want to drop this commit and do a full lint / include-what-you-use pass (I've also done one for appveyor but only added new includes / not removed useless ones unless blatantly obvious)